server/engine/src/main/java/org/eclipse/lsp/cobol/core/visitor/VisitorHelper.java

```diff
@@ -102,9 +102,11 @@ public static Locality getIntervalPosition(Locality start, Locality stop) {
    * @return the list of picture texts
    */
   public static List<String> retrievePicTexts(List<org.eclipse.lsp.cobol.core.CobolParser.DataPictureClauseContext> clauses) {
-    return clauses.stream()
-            .map(clause -> clause.getText().replaceAll(clause.getStart().getText(), "").trim())
-            .collect(toList());
+    List<String> list = new ArrayList<>(clauses.size());
+    for (CobolParser.DataPictureClauseContext clause : clauses) {
+      list.add(clause.getText().substring(clause.getStart().getText().length()).trim());
```

Comment:
Is this trying to extract just the `pictureString` or is the inclusion of the optional `IS` intentional?